### PR TITLE
feat(events): create new page for events

### PR DESCRIPTION
**Pull request by lirantal**
Creates a new page for consolidating events for the Node.js community
Related: [nodejs#2031](https://github.com/nodejs/nodejs.org/issues/2031)

Does the website markdown support tables? that would be much easier to display the events data both on the markdown as well as on the website.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
What about adding the title in 'site.json' so that we can directly link to that?

**Comment by ghost** [comment]
>Does the website markdown support tables? that would be much easier to display the events data both on the markdown as well as on the website.

It should support as what you do for md file, here's my preview in Visual Studio Code:
image

And if you run `npm run serve`, the final result will be:
image

**Comment by lirantal** [comment]
@Maledong added the events link to the site file
I'll also go ahead and push a change for the bullet-style to table-style

**Comment by ghost** [comment]
@lirantal：there's a nit——Please confirm that you haven't forgotten the '}' yet, because there's something wrong with the json file:
image

If you aren't sure whether your changes are right or not, you can `npm run test` and `npm run serve` to see the final result on your local machine :)

**Comment by lirantal** [comment]
Corrected, thanks!

**Comment by ghost** [comment]
@nodejs/documentation：Please have a review if you can, thanks!

**Review by willin — APPROVED** [review]
LGTM

File: locale/en/get-involved/node-events.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Node Events
```
Comment (ZYSzys) [inline locale/en/get-involved/node-events.md]:
```suggestion
title: Node.js Events
```

File: locale/en/get-involved/node-events.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+layout: contribute.hbs
+---
+
+# Node Events
```
Comment (ZYSzys) [inline locale/en/get-involved/node-events.md]:
```suggestion
# Node.js Events
```

File: locale/en/get-involved/node-events.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+This is a list of node events and conferences. Please submit a PR if you'd like to add a new event that isn't yet tracked in the list below.
```
Comment (ZYSzys) [inline locale/en/get-involved/node-events.md]:
```suggestion
This is a list of Node.js events and conferences. Please submit a PR if you'd like to add a new event that isn't yet tracked in the list below.
```

**Review by ZYSzys — COMMENTED** [review]
I'd like `node` => `Node.js` :)

Reference from your PR #2032

**Comment by lirantal** [comment]
Oh yes, good catch @ZYSzys 
Merged all the code suggestions!

**Review by ZYSzys — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by fhemberger — APPROVED** [review]

**Comment by ZYSzys** [comment]
Thank you ~

**Comment by ghost** [comment]
@nodejs/localization：Notice that 'Node Event' is newly added in the 'site.json' :)

**Pull request merged**